File: Babulsort/babulsort.go

package main

import "fmt"

func main(){
	numbers:=[]int{1, 20, 11, -40, 30, 2, 3}
	result:=Babul_sort(numbers)
	fmt.Println(result)

}

func Babul_sort(numbers []int)([]int){
	for i:=0;i<len(numbers);i++{
		for j:=0;j<len(numbers)-1;j++{
			if(numbers[j]>numbers[j+1]){
				temp:=numbers[j]
				numbers[j]=numbers[j+1]
				numbers[j+1]=temp
			}
		}
	}
	return numbers
}

File: Babulsort/babulsort.dart
void main(List<String> args) {
  var numbs = [1, 20, 11, -40, 30, 2, 3];
  var result = Babul_sort(numbs);
  print(result);


}
Babul_sort(number) {
    for (int i = 0; i < number.length; i++) {
      for (int j = 0; j < number.length - 1; j++) {
        if (number[j] > number[j + 1]) {
          var temp = number[j];
          number[j] = number[j + 1];
          number[j + 1] = temp;
        }
      }
    }
    return number;
  }
